modules/installation-about-custom.adoc

```diff
@@ -1,95 +1,51 @@
-
-[discrete]
-== Key concepts
-
-Before you complete a customized installation, review the following key concepts.
-
-[discrete]
-=== Targets
-
-The {product-title} installer operates on the notion of creating and destroying
-targets. Similar to other tools that operate on a graph of dependencies, like
-`make` and `systemd`, each target represents a subset of the dependencies in the
-graph. The main target in the installation program creates a cluster, but the other targets
-allow the user to interrupt this process and use or modify the intermediate
-artifacts, such as the the Kubernetes manifests that are installed into the
-cluster. Only the immediate dependencies of a target are written to disk by the
-installer, but you can run the installation program multiple times to update more
-dependencies.
-
-The installation program can create the following targets:
-
-`install-config`::
-The installation configuration contains the main parameters
-for the installation process. This configuration provides you with more options
-than the interactive prompts and comes pre-populated with default values.
-`manifest-templates`:: These are the unrendered Kubernetes manifest templates
-that `manifests` target uses. This target is unstable.
-`manifests`::
-You can generate the Kubernetes manifests that will be installed on the cluster.
-This target is unstable.
-`ignition-configs`::
-These files contain the Ignition configurations for the bootstrap, master, and
-worker machines.
-`cluster`::
-This target provisions the cluster and its associated infrastructure.
-
-The following targets can be destroyed by the installation program:
-
-`cluster`::
-This destroys the created cluster and its associated infrastructure.
-`bootstrap`::
-This destroys the bootstrap infrastructure.
-
-[discrete]
-=== Multiple invocations
-
-Because you must run the installation program to generate the assets that you need to
-customize your installation, you can run the installation program multiple times. The
-installation state is stored in a hidden file in the asset directory which 
-contains all of the intermediate artifacts. Storing the artifacts in this way
-lets you pause installation and modify the intermediate artifacts.
-
-For example, if you wanted to create a different number of worker machines, you
-must first run the installation program with the `install-config` target with the
-`openshift-install create install-config` command. After you provide the required
-parameters to the interactive installer, it writes the installation
-configuration into the target directory. You can then modify the installation
-configuration and run the installation program with the `cluster` target with the
-`openshift-install create cluster` command. The installation program reads the install
-configuration from disk, removes it from the target directory, and creates a
-cluster from the provided configuration.
+You can use the {product-title} installation program to customize four levels
```

Comment:
Because our [glossary](https://doc-stage.usersys.redhat.com/documentation/en-us/ccs_internal_documentation/1.0/html-single/glossary_of_terms_and_conventions_for_product_documentation/#i) says to not use "the installer" in product docs.

Reply:
> Because our glossary says to not use "the installer" in product docs.

Huh.  Do you know if the motivation for that is recorded anywhere?  I'm not sure if I should update the openshift/installer docs or not.

Reply:
That decision was made before my time, and I don't see a record for "why" in the word nerds notes.

Eh, I don't think that you have to change the openshift/installer docs because they won't feed into the official GA documentation.